packages/sqs/lib/sqs/AbstractSqsConsumer.ts

```diff
@@ -96,7 +97,8 @@ export abstract class AbstractSqsConsumer<
   >
   implements QueueConsumer
 {
-  private consumer?: Consumer
+  private consumers: Consumer[]
+  private readonly concurrentConsumersAmount: number
```

Comment:
🟢 Do we need to adjust `AbstractSnsSqsConsumer`? I don't know but asking just in case